### ISSUE TITLE
Front Panel was broken

### ISSUE THREAD

**Issue by chenm001** [LEAP-Core/leap#1]
all of components of Front Panel was broken, include hybrid-fp, front-panel-service, devtest, etc
I review the history, it broken by LLPI feature.

**Comment by michael-adler** [comment]
You are correct that the front panel is broken. We decided to stop supporting it a few years ago because it was a very old piece of code implemented in a manner no longer consistent with standard LEAP communication. Front panel functionality is replaced by other services like debug scan and stdio.

Is there something you wanted to do with the front panel?

**Comment by chenm001** [comment]
Sorry for delay, I just build leap example and evaluate it, but I found most example are not update to date.
